app/services/doc_auth_router.rb

```diff
@@ -38,6 +38,9 @@ module DocAuthRouter
     # i18n-tasks-use t('doc_auth.errors.general.no_liveness')
     DocAuth::Errors::GENERAL_ERROR =>
       'doc_auth.errors.general.no_liveness',
+    # i18n-tasks-use t('doc_auth.errors.dpi.top_msg')
```

Comment:
It is confusing. But [looking at the figma](https://www.figma.com/file/YejFLjUrlzZNjjM8McoudH/MVP%3A-Selfie-UX?type=design&node-id=1663-7228&mode=design&t=M0P4Jvz7f1pRFZb7-0), this was [where I found the same text](https://github.com/18F/identity-idp/blob/bfe1ac4e650d50e68119cec5458200eb050f710c/config/locales/doc_auth/en.yml#L74-L76).

Unsure if it should be renamed if it's really the default?

Reply:
> is only a few characters off from our default doc auth errors message

Given how complex this PR is, and the fact that these are basically the same I think we can combine them if that makes it simpler.

Reply:
@night-jellyfish , that's based on the assumption that we have `Failed` doc auth, but without any alert, so it's assumed to be image resolution `dpi` which makes it feel like a `general` error in some cases.

Adding `selfie` into this picture, we consider `Fail` is a larger scope, so technically we should only make that assumption when `doc_auth_result` failed.

Reply:
While thinking about this `metrics` error, potentially we can get rid of the check, since we use the `dpi` kind of as general error when nothing clear wrong there anyway.

Reply:
@charleyf - I keep going back and forth on this because on the one hand I see your point - it would simplify things! But on the other hand, I think it's separated so that if someone gets an error on only one side of their id, they don't get confused and think they have to re-do more than just the errored image. Do you think it's okay if we don't try to change that right now, and reconsider if it comes up as an issue again?

Reply:
@charleyf I know you're - OOO today, but since I received approval and am planning to merge without this comment being properly resolved, I wanted to leave a note. That note is that if it is something you'd like to pursue, I think (especially given the pending merge and that this is blocking other work) it would be best to be separately ticketed work. I hope that's okay!

Reply:
☑️ Merging this was the right call. This is the kind of small thing that absolutely shouldn't block this high priority work. Nice!